src/components/DriverOverview/DriverOverview.tsx

```diff
@@ -0,0 +1,208 @@
+    h2text: {
+        fontSize: 12,
+        paddingLeft: 10,
+        fontFamily: "Helvetica",
+    },
+    logoStyling: {
+        maxHeight: 60,
+        maxWidth: 60 * 1.7,
+        alignSelf: "center",
+        marginRight: 15,
+    },
+    warningSection: {
+        width: "100%",
+        paddingBottom: 10,
+    },
+    h3text: {
+        fontFamily: "Helvetica",
+        fontSize: 8,
+        paddingLeft: 15,
+        paddingTop: 5,
+    },
+    tableSection: {
+        width: "100%",
+        borderTop: "none",
+        borderBottom: "1px solid black",
+    },
+    tableRow: {
+        width: "100%",
+        fontSize: 8,
+        borderLeft: "1px solid black",
+        borderRight: "1px solid black",
+    },
+    tableColumn: {
+        padding: 5,
+        border: "1 solid black",
+        margin: 0.3,
+    },
+    flexColumn: {
+        flexDirection: "column",
+        display: "flex",
+    },
+    flexRow: {
+        flexDirection: "row",
+        display: "flex",
+        justifyContent: "space-between",
+    },
+    tableHeader: {
+        border: "1 solid black",
+        borderBottom: "none",
+        fontSize: 12,
+    },
+    nameColumnWidth: {
+        width: "15%",
+    },
+    addressColumnWidth: {
+        width: "20%",
+    },
+    contactColumnWidth: {
+        width: "15%",
+    },
+    packingDateColumnWidth: {
+        width: "15%",
+    },
+    parcelsColumnWidth: {
+        width: "10%",
+    },
+    instructionsColumnWidth: {
+        width: "25%",
+    },
+});
+
+const DriverOverviewCard: React.FC<DriverOverviewCardProps> = ({ driverName, date, data }) => {
+    const header = (
+        <View style={[styles.tableHeader, styles.flexRow]}>
+            <View style={[styles.tableColumn, styles.nameColumnWidth]}>
+                <Text>Name</Text>
+            </View>
+            <View style={[styles.tableColumn, styles.addressColumnWidth]}>
+                <Text>Address</Text>
+            </View>
+            <View style={[styles.tableColumn, styles.contactColumnWidth]}>
+                <Text>Contact</Text>
+            </View>
+            <View style={[styles.tableColumn, styles.packingDateColumnWidth]}>
+                <Text>Packing Date</Text>
+            </View>
+            <View style={[styles.tableColumn, styles.parcelsColumnWidth]}>
+                <Text>Parcels</Text>
+            </View>
+            <View style={[styles.tableColumn, styles.instructionsColumnWidth]}>
+                <Text>Instructions</Text>
+            </View>
+        </View>
+    );
+
+    const table = data.map((rowData, index) => {
+        return (
+            <View key={index} style={[styles.tableRow, styles.flexRow]} wrap={false}>
+                <View style={[styles.tableColumn, styles.nameColumnWidth]}>
+                    <Text>{rowData.name}</Text>
+                </View>
+                <View style={[styles.tableColumn, styles.addressColumnWidth]}>
+                    <Text>{rowData.address.line1}</Text>
```

Comment:
what happens if eg line2 is blank? does `<Text>` collapse to nothing or does it leave a blank line?

Reply:
The line collapses to nothing.